index.html

```diff
@@ -12,12 +12,22 @@
     <title>{{ site.title }}</title>
     <meta name="description" content="{{ site.description }}">
 
-	  <link rel="stylesheet" href="/assets/main.css">
+    <link rel="stylesheet" href="/assets/main.css">
     <link rel="canonical" href="{{ site.production_url }}{{ page.url | replace:'index.html',''}}">
     <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="/feed.xml">
-	  <link rel="stylesheet" href="/assets/tachyons.min.css"/>
-      <link href="https://fonts.googleapis.com/css?family=Open+Sans:600,700|Open+Sans+Condensed:300,700|Source+Sans+Pro:300,400,400i,600" rel="stylesheet">
-	</head>
+	<link rel="stylesheet" href="/assets/tachyons.min.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:600,700|Open+Sans+Condensed:300,700|Source+Sans+Pro:300,400,400i,600" rel="stylesheet">
+    <meta property="og:title" content="The First 100 Days And Counting"/>
+    <meta property="og:type" content="article"/>
+    <meta property="og:url" content="https://100days.envirodatagov.org/index.html"/>
```

Comment:
A couple lines above, I used this pattern to string index.html from a url (which is also intended to be canonical), so we could use the same thing here:
```
{{ site.production_url }}{{ page.url | replace:'index.html',''}}
```